lib/l10n.js

```diff
@@ -13,456 +13,569 @@ var l10n = {
+        es: "Por favor ingrese su género como masculino o femenino",
+        ko: "성별을 입력 해 주세요"
     },
     error_validate_military_or_overseas: {
         en: "Sorry, I didn't get that. Please answer (military/overseas/no)",
-        es: "Lo siento, no entendí eso. Por favor contesta (militar/extranjero/no)"
+        es: "Lo siento, no entendí eso. Por favor contesta (militar/extranjero/no)",
+        ko: "유효한 입력이 아닙니다. 군인이면 militar, 해외 투표는 overseas, 둘 다 아니면 no 를 입력 해 주세요."
     },
     error_validate_ssn: {
         en: 'Please enter your social security number like 123-45-6789',
-        es: "Por favor ingrese su número de seguro social como 123-45-6789"
+        es: "Por favor ingrese su número de seguro social como 123-45-6789",
+        ko: "소셜 시큐리티 번호를 000-00-0000 형식으로 입력 해 주세요"
     },
     error_validate_ssn_last4: {
         en: 'Please enter just the last 4 digits of your social security number',
-        es: "Por favor ingrese solo los ultimos 4 digitos de su número de seguro social."
+        es: "Por favor ingrese solo los ultimos 4 digitos de su número de seguro social.",
+        ko: "소셜 시큐리티 번호의 마지막 4자리만을 입력 해 주세요."
     },
     error_validate_state_id_number: {
         en: 'Please enter a valid {{settings.state}} ID. If you don\'t have one, reply "NONE" and we will try something else.',
-        es: "Por favor ingrese una identificación de {{settings.state}} válida. Si no tiene una, yo no puedo entregar su inscripción. Responde NINGUNO y yo le enviaré un formulario que puede imprimir y mandar por correo."
+        es: "Por favor ingrese una identificación de {{settings.state}} válida. Si no tiene una, yo no puedo entregar su inscripción. Responde NINGUNO y yo le enviaré un formulario que puede imprimir y mandar por correo.",
+        ko: "유효한 {{settings.state}} 주 정부 신분증 번호를 입력하세요. 주 신분증이 없으면 none 이라고 입력하세요."
     },
     error_state_deadline_expired: {
         en: 'Sorry, the online voter registration deadline for {{settings.state}} has passed.',
-        es: "Lo siento, la fecha límite de inscripción para votar en {{settings.state}} ha pasado."
+        es: "Lo siento, la fecha límite de inscripción para votar en {{settings.state}} ha pasado.",
+        ko: "{{settings.state}} 주의 유권자 등록 마감일이 지났습니다."
     },
     error_validate_state_abbreviation: {
         en: 'Sorry, that\'s not a valid US state abbreviation.',
-        es: "Lo siento, eso no es un abreviatura de estado válido."
+        es: "Lo siento, eso no es un abreviatura de estado válido.",
+        ko: "그"
     },
     error_validate_state_name: {
         en: 'Sorry, that\'s not a valid US state name.',
-        es: "Lo siento, eso no es un nombre de estado válido."
+        es: "Lo siento, eso no es un nombre de estado válido.",
+        ko: "유효한 주 이름이 아닙니다"
     },
     msg_happy_birthday: {
         en: 'Happy birthday! \u{1F382}',
-        es: "¡Feliz cumpleaños! \u{1F382}"
+        es: "¡Feliz cumpleaños! \u{1F382}",
+        ko: "생일 축하합니다!"
```

Comment:
for consistency, please include the unicode birthday cake emoji in this message
🎂

Reply:
Adding